extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -289,7 +289,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @return {!Promise}
    */
   maybeInitializeSessionManager_() {
-    if (this.config_[PERSIST_EVENT_TIMESTAMP] && this.type_) {
+    if (this.config_[PERSIST_SESSION_VALUE] && this.type_) {
```

Comment:
Since we only check `PERSIST_SESSION_VALUE` once, let's remove it and determine its value in this function instead.